src/components/books/Books.js

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchBooks } from '../../redux/books/books';
 import Book from '../book/Book';
 import AddNewBook from '../form/AddNewBook';
 
 function Books() {
+  const dispatch = useDispatch();
+
   // GET books from redux store
-  const books = useSelector((state) => state.booksReducer.books);
+  const books = useSelector((state) => state.booksReducer);
+
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
```

Comment:
- [ ] Good job on  adding, deleting, and fetching your books to the API but when rendering your books there is a bit of issue, When I go to the categories page it works fine I see the message `under construction but when I navigate back to books they duplicate themselves but if I reload the page the duplication disappears, Kindly have a look at that video and fix the issue :point_down: 

https://user-images.githubusercontent.com/75126481/150427358-5763ff86-1b3f-4a65-9597-f29a164c373c.MP4